File: Go/session-20-capstone/examples/counter.go
package main

type Counter struct {
	n int
}

func (c *Counter) Inc() int {
	c.n++
	return c.n
}

func (c *Counter) Value() int {
	return c.n
}

func main() {
	c := &Counter{}
	c.Inc()
	_ = c.Value()
}

File: PHP/session-15-session-cookie/examples/counter.php
<?php
declare(strict_types=1);

session_set_cookie_params([
    'lifetime' => 0,
    'path' => '/',
    'secure' => false,
    'httponly' => true,
    'samesite' => 'Lax',
]);
session_start();

$_SESSION['visits'] = (int)($_SESSION['visits'] ?? 0) + 1;
header('Content-Type: text/html; charset=UTF-8');
echo '<p>بازدید: ' . (int)$_SESSION['visits'] . '</p>';
echo '<p><a href="counter.php">رفرش</a></p>';
